agent/api/statechange_test.go

```diff
@@ -21,14 +21,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/aws/amazon-ecs-agent/agent/api/serviceconnect"
-
 	apicontainer "github.com/aws/amazon-ecs-agent/agent/api/container"
 	apicontainerstatus "github.com/aws/amazon-ecs-agent/agent/api/container/status"
+	"github.com/aws/amazon-ecs-agent/agent/api/serviceconnect"
```

Comment:
Not sure if this was meant to be here, I think it might have been like a rebase/merge artifact but I'm not sure it is going to impact anything so meh.

Reply:
no, I just corrected the import order, moved it into the `amazon-ecs-agent` import group from above.